### PR TITLE
Add 3.6 and fix typo

### PR DESCRIPTION
**Pull request by hugovk**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -77,7 +77,7 @@ <h1 id="logo"><b><i>Py</i>Format</b> Using <i>%</i> and <i>.format()</i> for gre
                         With this site we try to show you the most common use-cases covered by the <a href="https://docs.python.org/2/library/stdtypes.html#string-formatting">old</a> and <a href="https://docs.python.org/3/library/string.html#string-formatting">new</a> style string formatting API with practical examples.
                     </p>
                     <p>
-                        All examples on this page work out of the box with with Python 2.7, 3.2, 3.3, 3.4, and 3.5 without requiring any additional libraries.
+                        All examples on this page work out of the box with with Python 2.7, 3.2, 3.3, 3.4, 3.5 and 3.6 without requiring any additional libraries.
```
Comment (hugovk) [inline templates/index.html]: TODO Tidy up "with with".

**Comment by cclauss** [comment]
3.7?

**Comment by hugovk** [comment]
Thanks, changed it to 3.2+.

**Comment by hugovk** [comment]
Ping @ulope.

**Comment by cclauss** [comment]
"Explicit is better than implicit"  ;-)

**Pull request closed without merging**